server/src/mds3.gdc.js

```diff
@@ -1171,6 +1173,13 @@ function mayApplyBinning(samples, twLst) {
 			}
 			const bins = compute_bins(tw.q, p => summary)
 
+			/* sql CTE returns {name,label}, where compute_bins does not return name
```

Comment:
@siosonel @congyu-lu CTE generates both `label` and `name` attributes for each bin, with same value. what's reason for it? can it just use `label` but not `name`?